core/templates/dev/head/services/contextual/url.service.spec.ts

```diff
@@ -37,12 +38,13 @@ describe('Url Service', () => {
     };
 
     urlService = TestBed.get(UrlService);
-    spyOn(urlService, 'getCurrentLocation').and.returnValue(mockLocation);
+    windowRef = TestBed.get(WindowRef);
+    spyOnProperty(windowRef, 'nativeWindow').and.callFake(() => ({
+      location: mockLocation}));
   });
 
   it('should return correct query value list for each query field', () => {
     expect(urlService.getQueryFieldValuesAsList('field1')).toEqual([]);
-
```

Comment:
Why is this deleted? (I think we can leave it as it separates the beaseline verification and actions part of the test.

Reply:
Done.